From: zhong jiang <zhongjiang@huawei.com>
Date: Wed, 28 Nov 2018 23:04:48 -0800
Subject: [PATCH 01/19] net: ethernet: remove redundant include
Origin: https://git.kernel.org/linus/e641e99f261f5203a911a9e0db54a214460d2cc4

Manual cherry-pick from e641e99f261f5203a911a9e0db54a214460d2cc4:

    module.h already contained moduleparam.h,  so it is safe to remove
    the redundant include.

    The issue is detected with the help of Coccinelle.

    Signed-off-by: zhong jiang <zhongjiang@huawei.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

limited only to the amazon/ena driver

Signed-off-by: Noah Meyerhans <noahm@debian.org>
---
 drivers/net/ethernet/amazon/ena/ena_netdev.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/ethernet/amazon/ena/ena_netdev.c b/drivers/net/ethernet/amazon/ena/ena_netdev.c
index d906293ce07d..69a49784b204 100644
--- a/drivers/net/ethernet/amazon/ena/ena_netdev.c
+++ b/drivers/net/ethernet/amazon/ena/ena_netdev.c
@@ -39,7 +39,6 @@
 #include <linux/if_vlan.h>
 #include <linux/kernel.h>
 #include <linux/module.h>
-#include <linux/moduleparam.h>
 #include <linux/numa.h>
 #include <linux/pci.h>
 #include <linux/utsname.h>
-- 
2.19.2

